### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Obertg3** [mriscoc/Ender3V2S1#659] [Q&A]
Hello I cant get any firmware to work on my ender 3v2 after installing sprite pro kit I was using jyers but now none of them work and it stays on the creality logo, any help would be greatly appreciated and even offer money to anyone who can get it to work thanks

**Comment by TheWebMachine** [comment]
Try (re-)flashing the assets/firmware for the display, itself.

**Reply by Obertg3** [reply]
> Try (re-)flashing the assets/firmware for the display, itself. 

I tried that using the gia dwin set on mirscoc nothing happens I've tried all of the precompiled .bin on here and a bunch of others

**Reply by TheWebMachine** [reply]
Which screen do you have, exactly?

**Reply by Obertg3** [reply]
I got it had to go to three different stores to get a new sd card thanks but I've got everything going now except the sprite is skipping and not extruding anything I set the e steps to 429.4 like the manual said any help on how to adjust the tension or e steps would be amazing also the y axis is a little bit too far is there anyway to set the y home

**Reply by Obertg3** [reply]
> Which screen do you have, exactly?

Please help

**Comment by mriscoc** [comment]
Check if you have a STM32 or GD32 chip

**Reply by Obertg3** [reply]
I got it had to go to three different stores to get a new sd card thanks but I've got everything going now except the sprite is skipping and not extruding anything I set the e steps to 429.4 like the manual said any help on how to adjust the tension or e steps would be amazing also the y axis is a little bit too far is there anyway to set the y home

**Reply by Obertg3** [reply]
Pls help

**Reply by mriscoc** [reply]
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
Reduce the extruder max speed and acceleration.

**Reply by Obertg3** [reply]
> https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
> 
> Reduce the extruder max speed and acceleration.

THANKYOU. Lowered acc and max speed replaced nozzle and leveld multiple times and it's working great now, I will probably just wait to get bl touch to use the mesh since it's off centered did the four corners and center Thankyou
